src/content/docs/faq.mdx

```diff
@@ -246,6 +246,12 @@ command works.
 [Uninstall](/uninstall) has details on how to remove Shorebird from your
 system should you choose.
 
+### Can the user delete patches from their device?
+
+Yes. Since the patches are store in the application data folder, if users
+delete the application data in their devices, the patches will deleted as well.
+This will make the application be sent to the original state of the release.
```

Comment:
I think I'd provide a little more context here (specifically, the steps the user needs to go through to do this). As it's written, this makes it sound like a very easy and obvious thing for a user to do.

Reply:
Tweaked a little bit, can you take another look please?